web/requirements.txt

```diff
@@ -1,8 +1,9 @@
 boto==2.39.0
 dj-database-url==0.4.0
-Django==1.9.2
+Django==1.10.1
+django-cors-middleware==1.3.1
```

Comment:
You can read the file if you want. I did to make sure it was satisfactorily simple and secure. This is a fork from the django-cors-headers project which is dead. This is a little bit on life support because they're just using a mixin provided for people who are using middleware classes that have a deprecated style. With the next django they need to have middleware that actually uses the new style. But since it's so small we could feasibly fork it if we had to and convert ourselves. Either way we need cors in order to allow people on other web pages to access our code.